addons/website_mass_mailing/views/snippets_templates.xml

```diff
@@ -80,17 +92,16 @@
         <div data-js="mailing_list_subscribe"
             t-att-data-selector="selector"
             t-attf-data-exclude=".o_newsletter_modal #{selector}">
-            <we-button data-select_mailing_list="" data-no-preview="true">Change Newsletter</we-button>
+            <we-select class="o_we_inline" string="Change Newsletter" data-name="mailing_list" data-no-preview="true"></we-select>
         </div>
         <div data-js="recaptchaSubscribe"
             t-att-data-selector="selector">
             <t t-set="recaptcha_public_key" t-value="request.env['ir.config_parameter'].sudo().get_param('recaptcha_public_key')"/>
             <we-checkbox t-if="recaptcha_public_key" string="Show reCaptcha Policy" data-toggle-recaptcha-legal="" data-no-preview="true"/>
         </div>
         <div t-att-data-selector="selector" data-drop-near="p, h1, h2, h3, blockquote, .card"/>
-        <div data-js="newsletter_popup"
-            data-selector=".o_newsletter_popup">
-            <we-button data-select_mailing_list="" data-no-preview="true">Change Newsletter</we-button>
+        <div data-js="newsletterPopup" data-selector=".o_newsletter_popup">
+            <we-select class="o_we_inline" string="Change Newsletter" data-name="mailing_list" data-no-preview="true"></we-select>
```

Comment:
This should have been removed and share the previous option (so remove the data-exclude)